pkg/ebpf/tracee.go

```diff
@@ -689,11 +697,84 @@ func (t *Tracee) populateBPFMaps() error {
 				return err
 			}
 		}
+		if e == PrintSyscallTableEventID {
```

Comment:
There is too much code here.
Find a way to export some of it to a sub-function.

Reply:
I exported the `syscallList` check to a sub-function, it still needs to be reduced, after your PR will merge, ill change the place of the 'sys_call_table' address to your map. then its should be fine (: